source/2020-06-19-the-ember-times-issue-153.md

```diff
@@ -14,14 +14,13 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Set up fast with `ember-new-plus` ⚡️](https://github.com/atej/ember-new-plus)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Aseem Taneja (@atej)](https://github.com/atej) has created a bash script that automates the setup of some often-used tools in an ember project. To further enhance your DX, it also recommends [VS Code](https://code.visualstudio.com/) extensions and generates settings to integrate with them.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Based on your selections, you get – linting and formatting with [ESLint](https://eslint.org) + [Prettier](https://prettier.io), utility-first CSS with [Tailwind CSS](https://tailwindcss.com) + [PostCSS](https://postcss.org), git hooks with [Husky](https://github.com/typicode/husky), pre-commit linting of staged files with [lint-staged](https://github.com/okonet/lint-staged), adherance to [conventional commit messages](https://conventionalcommits.org) with [commitlint](https://github.com/conventional-changelog/commitlint), and a launch configuration for in-editor debugging with VS Code's [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome).
+
+The configuration files generated when you first [run the script](https://github.com/atej/ember-new-plus#quickstart-) can be tuned to your preference later.
```

Comment:
I noticed that we currently only have 1 direct link to your GitHub repo (the title, which can be missed by readers). I think it'd be good to provide the link once more at the end.

Something like:
```suggestion
The configuration files generated when you first [run the script](https://github.com/atej/ember-new-plus#quickstart-) can be tuned to your preference later. If you are about to make a new Ember project, give [ember-new-plus](https://github.com/atej/ember-new-plus) a try!
```
?